Makefile

```diff
@@ -48,7 +48,7 @@ LDFLAGS := "-X main.version=$(VERSION) \
 
 .DEFAULT_GOAL := build
 .PHONY: build
-build: manifests generate kubectl-gadget gadget-default-container
+build: kubectl-gadget gadget-default-container
```

Comment:
These are defined in crd.mk, probably we want to keep them here and add documentation for them too.